Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 name = "multibase"
 version = "0.7.0"
-description = "multibase rust"
-homepage = "https://github.com/multiformats/rust-multibase"
-authors = ["Friedel Ziegelmayer <dignifiedquire@gmail.com>"]
+authors = [
+    "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
+    "Qinxuan Chen <koushiro.cqx@gmail.com>",
```

Comment:
I'd like to reserve the "authors" for people that have also permission to publish a crate. This should be an entry point for people that want to get in touch with the maintainers.